app/src/main/res/values/strings.xml

```diff
@@ -266,6 +266,10 @@
     <!-- wallet info -->
     <string name="wallet_info_my_profile">My Profile</string>
     <string name="wallet_info_qr_code_desc">Transacting in person? Others can scan this QR code from the Tari Aurora App to send you tXTR.</string>
+    <string name="wallet_info_yat_disconnected">Disconnected</string>
```

Comment:
Where are you using `wallet_info_yat_disconnected` and `wallet_info_yat_connected` strings?

Reply:
Removed, it was just my glitch